RoslyJump/Properties/AssemblyInfo.cs

```diff
@@ -5,11 +5,11 @@
 // General Information about an assembly is controlled through the following 
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("RoslyJump")]
+[assembly: AssemblyTitle("RoslyJump.VSIX_2019")]
```

Comment:
Not sure it should be changed.

Reply:
Decided to leave it as `RoslyJump.VSIX_2019` because it will implicitly change output extension name to `RoslyJump.VSIX_2019.vsix` which may reduce confusion.